docs/CLI.md

```diff
@@ -100,7 +100,9 @@ jest --update-snapshot --detectOpenHandles
 
 _Note: CLI options take precedence over values from the [Configuration](Configuration.md)._
 
-<AUTOGENERATED_TABLE_OF_CONTENTS>
+import TOCInline from "@theme/TOCInline"
+
+<TOCInline toc={toc[toc.length - 1].children}/>
```

Comment:
`toc[toc.length - 1].children` is somewhat weird. I don't have any suggestion for an improvement, but this seems magical and somewhat manual at the same time 😅

Reply:
That may be clearer if you read this doc :)

https://v2.docusaurus.io/docs/next/markdown-features/inline-toc/#custom-table-of-contents

Basically, you can pass whatever you want from this component. You could have written `toc={toc}` , but we don't want the full page TOC, we just want to list the children's of the last item's level-2 heading (reference).

see how the right toc and the inlined toc content is slightly different

![image](https://user-images.githubusercontent.com/749374/105499984-8afa4380-5cc2-11eb-9fe8-e4bd9f49eed2.png)